src/restart_analysis_and_reload_templates.py

```diff
@@ -0,0 +1,43 @@
+import logging
+import signal
+import sys
+
+
+from helperFunctions.program_setup import program_setup
+from scheduler.Analysis import AnalysisScheduler
+
+
+PROGRAM_NAME = 'Restart FACT Analysis Component'
+PROGRAM_DESCRIPTION = 'Restart FACT Analysis Component and reload Analysis Templates'
+
+
+def shutdown(signum, frame):
+    global run
+    logging.info('shutting down FACT Analysis Component')
+    run = False
+
+
+signal.signal(signal.SIGINT, shutdown)
+signal.signal(signal.SIGTERM, shutdown)
+
+
+def main():
```

Comment:
The command_line_options have to be given as parameter (and propagated to program_setup).